src/library/scala/collection/Iterator.scala

```diff
@@ -262,14 +256,13 @@ trait Iterator[+A] extends IterableOnce[A] with IterableOnceOps[A, Iterator, Ite
       else go(step)
     }
 
-    def hasNext = filled || fill()
+    def hasNext: Boolean =
+      filled || fill()
```

Comment:
again, not sure why on a new line. you're doing a whole bunch of reducing the number of lines in other places, but then add new lines here. this is a nitpick—I'm just confused

Reply:
because the method "hasNext" line.